# Greylist information

In this file we collect information about which tests are currently
greylisted. Please add a reason and date, and possibly links to issues
or PRs.

 - `tests/js/server/resilience/repair-distribute-shards-like-spec.js`

 Test "resilience"
    [FAILED]  tests/js/server/resilience/repair-distribute-shards-like-spec.js

      "Collections with distributeShardsLike with data if newly created, should always be ok" failed: AssertionError: expected false to be true
    at Context.<anonymous> (tests/js/server/resilience/repair-distribute-shards-like-spec.js:526:61)
    at Function.global.DEFINE_MODULE.exports.nextTick [as immediately] (js/common/bootstrap/modules/process.js:30:5)

      "Collections with distributeShardsLike with data if broken, should be repaired" failed: AssertionError: expected false to be true
    at createBrokenClusterState (tests/js/server/resilience/repair-distribute-shards-like-spec.js:220:57)
    at Context.<anonymous> (tests/js/server/resilience/repair-distribute-shards-like-spec.js:560:11)

      "Collections with distributeShardsLike with data if interrupted, should complete repairs" failed: AssertionError: expected false to be true
    at createBrokenClusterState (tests/js/server/resilience/repair-distribute-shards-like-spec.js:220:57)
    at Context.<anonymous> (tests/js/server/resilience/repair-distribute-shards-like-spec.js:624:13)

      "Collections with distributeShardsLike with data if called via GET, only return planned operations" failed: AssertionError: expected false to be true
    at createBrokenClusterState (tests/js/server/resilience/repair-distribute-shards-like-spec.js:220:57)
    at Context.<anonymous> (tests/js/server/resilience/repair-distribute-shards-like-spec.js:761:11)

   Suites failed: 1 Tests Failed: 4

 - `tests/js/server/shell/shell-foxx-legacy-model.js` fails in docker

    [FAILED]  tests/js/server/shell/shell-foxx-legacy-model.js

      "testFromShaped" failed: ArangoError 4: internal error - ArangoError: internal error
    at Object.flush (/work/ArangoDB/js/server/bootstrap/modules/internal.js:137:31)
    at Object.testFromShaped (tests/js/server/shell/shell-foxx-legacy-model.js:332:20)
    at Object.run (/work/ArangoDB/js/common/modules/jsunity/jsunity.js:515:23)
    at Object.Run [as run] (/work/ArangoDB/js/common/modules/jsunity.js:171:24)
    at tests/js/server/shell/shell-foxx-legacy-model.js:473:9
    at tests/js/server/shell/shell-foxx-legacy-model.js:477:3
    at RunTest (/work/ArangoDB/js/common/modules/jsunity.js:248:12)
    at eval (eval at callback (/work/ArangoDB/js/actions/_admin/app.js:82:18), <anonymous>:2:52)
    at eval (eval at callback (/work/ArangoDB/js/actions/_admin/app.js:82:18), <anonymous>:2:132)
    at Function.callback (/work/ArangoDB/js/actions/_admin/app.js:82:18)

      "testFromWal" failed: ArangoError 4: internal error - ArangoError: internal error
    at Object.flush (/work/ArangoDB/js/server/bootstrap/modules/internal.js:137:31)
    at Object.testFromWal (tests/js/server/shell/shell-foxx-legacy-model.js:347:20)
    at Object.run (/work/ArangoDB/js/common/modules/jsunity/jsunity.js:515:23)
    at Object.Run [as run] (/work/ArangoDB/js/common/modules/jsunity.js:171:24)
    at tests/js/server/shell/shell-foxx-legacy-model.js:473:9
    at tests/js/server/shell/shell-foxx-legacy-model.js:477:3
    at RunTest (/work/ArangoDB/js/common/modules/jsunity.js:248:12)
    at eval (eval at callback (/work/ArangoDB/js/actions/_admin/app.js:82:18), <anonymous>:2:52)
    at eval (eval at callback (/work/ArangoDB/js/actions/_admin/app.js:82:18), <anonymous>:2:132)
    at Function.callback (/work/ArangoDB/js/actions/_admin/app.js:82:18)

      "testFromDbWithSchema" failed: ArangoError 4: internal error - ArangoError: internal error
    at Object.flush (/work/ArangoDB/js/server/bootstrap/modules/internal.js:137:31)
    at Object.testFromDbWithSchema (tests/js/server/shell/shell-foxx-legacy-model.js:364:20)
    at Object.run (/work/ArangoDB/js/common/modules/jsunity/jsunity.js:515:23)
    at Object.Run [as run] (/work/ArangoDB/js/common/modules/jsunity.js:171:24)
    at tests/js/server/shell/shell-foxx-legacy-model.js:473:9
    at tests/js/server/shell/shell-foxx-legacy-model.js:477:3
    at RunTest (/work/ArangoDB/js/common/modules/jsunity.js:248:12)
    at eval (eval at callback (/work/ArangoDB/js/actions/_admin/app.js:82:18), <anonymous>:2:52)
    at eval (eval at callback (/work/ArangoDB/js/actions/_admin/app.js:82:18), <anonymous>:2:132)
    at Function.callback (/work/ArangoDB/js/actions/_admin/app.js:82:18)

      "testSettingFromDb" failed: ArangoError 4: internal error - ArangoError: internal error
    at Object.flush (/work/ArangoDB/js/server/bootstrap/modules/internal.js:137:31)
    at Object.testSettingFromDb (tests/js/server/shell/shell-foxx-legacy-model.js:379:20)
    at Object.run (/work/ArangoDB/js/common/modules/jsunity/jsunity.js:515:23)
    at Object.Run [as run] (/work/ArangoDB/js/common/modules/jsunity.js:171:24)
    at tests/js/server/shell/shell-foxx-legacy-model.js:473:9
    at tests/js/server/shell/shell-foxx-legacy-model.js:477:3
    at RunTest (/work/ArangoDB/js/common/modules/jsunity.js:248:12)
    at eval (eval at callback (/work/ArangoDB/js/actions/_admin/app.js:82:18), <anonymous>:2:52)
    at eval (eval at callback (/work/ArangoDB/js/actions/_admin/app.js:82:18), <anonymous>:2:132)
    at Function.callback (/work/ArangoDB/js/actions/_admin/app.js:82:18)

      "testSettingFromDbWithSchema" failed: ArangoError 4: internal error - ArangoError: internal error
    at Object.flush (/work/ArangoDB/js/server/bootstrap/modules/internal.js:137:31)
    at Object.testSettingFromDbWithSchema (tests/js/server/shell/shell-foxx-legacy-model.js:401:20)
    at Object.run (/work/ArangoDB/js/common/modules/jsunity/jsunity.js:515:23)
    at Object.Run [as run] (/work/ArangoDB/js/common/modules/jsunity.js:171:24)
    at tests/js/server/shell/shell-foxx-legacy-model.js:473:9
    at tests/js/server/shell/shell-foxx-legacy-model.js:477:3
    at RunTest (/work/ArangoDB/js/common/modules/jsunity.js:248:12)
    at eval (eval at callback (/work/ArangoDB/js/actions/_admin/app.js:82:18), <anonymous>:2:52)
    at eval (eval at callback (/work/ArangoDB/js/actions/_admin/app.js:82:18), <anonymous>:2:132)
    at Function.callback (/work/ArangoDB/js/actions/_admin/app.js:82:18)
 
 - `./tests/js/server/replication/fuzz/replication-fuzz.js` often fails with

    [FAILED]  tests/js/server/replication/fuzz/replication-fuzz.js

      "testFuzz" failed: at assertion #5: assertEqual: (edge1lPYZAhDaHTXARyb1553464982839-7-6243638505965989336primary-["_key"]edge-["_from","_to"]fuchs5NcDJ5W7op5FaurO1553464981502-2994-2799813630684851127primary-["_key"]fuchsdIE7aXAEAJNemsAo1553465047639-2006-12100265488340768412primary-["_key"]hash-["qSO6vCeQ0NFNyRqb1553464994849"]skiplist-["VrXYo1SlNE8Uzf141553465023423"]fuchsWhIJSLoR4ohvwETO1553465028502-6-2495317728447060923primary-["_key"]testErZaaOqAixhMcViW1553465028081-3-1834778075040168125primary-["_key"]) is not equal to (edge1lPYZAhDaHTXARyb1553464982839-0-0primary-["_key"]edge-["_from","_to"]fuchs5NcDJ5W7op5FaurO1553464981502-2994-2799813630684851127primary-["_key"]fuchsdIE7aXAEAJNemsAo1553465047639-2006-12100265488340768412primary-["_key"]fuchsWhIJSLoR4ohvwETO1553465028502-6-2495317728447060923primary-["_key"]testErZaaOqAixhMcViW1553465028081-0-0primary-["_key"])
(Error
    at assertEqual (/work/ArangoDB/js/common/modules/jsunity/jsunity.js:103:19)
    at /work/ArangoDB/tests/js/server/replication/replication-fuzz.js:506:11
    at compare (/work/ArangoDB/tests/js/server/replication/replication-fuzz.js:141:5)
    at Object.testFuzz (/work/ArangoDB/tests/js/server/replication/replication-fuzz.js:183:7)
    at Object.run (/work/ArangoDB/js/common/modules/jsunity/jsunity.js:515:23)
    at Object.Run [as run] (/work/ArangoDB/js/common/modules/jsunity.js:171:24)
    at /work/ArangoDB/tests/js/server/replication/replication-fuzz.js:518:9
    at /work/ArangoDB/tests/js/server/replication/replication-fuzz.js:521:3
    at RunTest (/work/ArangoDB/js/common/modules/jsunity.js:248:12)
    at /work/ArangoDB/js/common/modules/@arangodb/testrunner.js:47:13)

   Suites failed: 1 Tests Failed: 1

 - `./tests/js/server/recovery/corrupted-crc-mmfiles-grey.js` sometimes fails with

      [FAILED]  tests/js/server/recovery/corrupted-crc-mmfiles.js

      "test" failed: failed to read /work/tmp/arangosh_MNeeKL/crashtmp/6/tmp/testresult.json - ArangoError 2: No such file or directory: while reading /work/tmp/arangosh_MNeeKL/crashtmp/6/tmp/testresult.json

 - `./tests/js/client/shell/shell-foxx-api-spec-grey.js` sometimes fails with

    [FAILED]  tests/js/client/shell/shell-foxx-api-spec.js

      "Foxx service dependencies should be overwritten after replace" failed: ArangoError: service files missing
Mount: /foxx-crud-test
    at Object.exports.checkRequestResult (js/client/modules/@arangodb/arangosh.js:97:21)
    at Object.install (js/client/modules/@arangodb/foxx/manager.js:282:12)
    at Context.it (tests/js/client/shell/shell-foxx-api-spec.js:1001:17)

 - `./tests/js/client/resilience/foxxmaster.js` sometimes fails with

    [FAILED]  tests/js/client/resilience/foxxmaster.js

      "testQueueFailoverDuringJob" failed: Error: Supervision should have moved the Foxx queues and Foxx queues should have been started to run on a new coordinator - Error: Supervision should have moved the Foxx queues and Foxx queues should have been started to run on a new coordinator
    at Object.testQueueFailoverDuringJob (/work/ArangoDB/tests/js/client/resilience/foxxmaster.js:252:15)
    at Object.run (/work/ArangoDB/js/common/modules/jsunity/jsunity.js:515:23)
    at Object.Run [as run] (/work/ArangoDB/js/common/modules/jsunity.js:171:24)
    at /work/ArangoDB/tests/js/client/resilience/foxxmaster.js:263:9
    at /work/ArangoDB/tests/js/client/resilience/foxxmaster.js:266:3
    at RunTest (/work/ArangoDB/js/common/modules/jsunity.js:248:12)
    at /work/ArangoDB/js/common/modules/@arangodb/testrunner.js:47:13
    at arrayEach (/work/ArangoDB/js/node/node_modules/lodash/lodash.js:537:11)
    at Function.forEach (/work/ArangoDB/js/node/node_modules/lodash/lodash.js:9359:14)
    at runJSUnityTests (/work/ArangoDB/js/common/modules/@arangodb/testrunner.js:38:5)
